.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - develop
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - develop
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose
+    - name: Run tests
+      run: cargo test --verbose
+
```

Comment:
Add one last step here that does `cargo install .` to verify a non-debug build is successful (and properly installs).

Reply:
Thanks for the review.

`cargo install` step added. New issue created here: #25

Reply:
:heavy_check_mark: